lib/modules/platform/github/index.ts

```diff
@@ -1176,23 +1175,22 @@ export async function setBranchStatus({
 
 /* istanbul ignore next */
 async function getIssues(): Promise<Issue[]> {
-  const result = await githubApi.queryRepoField<Issue>(
-    getIssuesQuery,
-    'issues',
-    {
-      variables: {
-        owner: config.repositoryOwner,
-        name: config.repositoryName,
-        user: config.renovateUsername,
-      },
-    },
-  );
-
-  logger.debug(`Retrieved ${result.length} issues`);
-  return result.map((issue) => ({
-    ...issue,
+  const result: Issue[] = (
+    await githubApi.getJson<Issue[]>(
+      `repos/${config.parentRepo ?? config.repository}/issues?creator=${
+        config.renovateUsername
+      }&state=all`,
```

Comment:
we should add sorting, otherwise we get newest created issues and pulls first. I'm not sure how the etag will work. we probably fetch all issues on every run because we or another user changed a pull.

🤔

Reply:
It's going to be unlikely to have >100 issues created by Renovate in a repo